README.md

```diff
@@ -176,6 +176,54 @@ E.g.,
 $ poetry run pytest todo_app/data/classes/test_view_model.py::test_view_model_done_items_property_returns_items_with_status_done
 ```
 
+### Docker
+
+#### Setup
+
+From the project root (`DevOps-Course-Starter`), run the following from your preferred shell:
+
+```shell
+$ docker build --target test --tag <name_you_want_to_give_to_test_image> .
+```
+
+#### Run all unit and integration tests
+
+From the project root (`DevOps-Course-Starter`), run the following from your preferred shell:
+
+```shell
+$ docker run --env-file .env.test <name_you_gave_to_test_image_during_setup> 
```

Comment:
I understand that this gives people more flexibility, but personally I'd just give the image a name and assume it here, to make allowing people to copy-paste instructions easier!